python/MooseDocs/extensions/appsyntax.py

```diff
@@ -468,8 +468,11 @@ def defaultSettings():
     def createTokenFromSyntax(self, parent, info, page, obj):
 
         primary = SyntaxList(None, **self.attributes)
+        if self.settings['groups']:
```

Comment:
@joshuahansel @cbolisetti I am hoping this fixes the syntax list problems you are seeing in your apps